app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewData.kt

```diff
@@ -268,13 +268,13 @@ class OverviewData @Inject constructor(
 
     var maxRatioValueFound = 5.0                    //even if sens data equals 0 for all the period, minimum scale is between 95% and 105%
     var minRatioValueFound = -maxRatioValueFound
-    val ratioScale = Scale()
+    var ratioScale = Scale()
     var ratioSeries: LineGraphSeries<ScaledDataPoint> = LineGraphSeries()
 
     var maxFromMaxValueFound = Double.MIN_VALUE
     var maxFromMinValueFound = Double.MIN_VALUE
-    val dsMaxScale = Scale()
-    val dsMinScale = Scale()
+    var dsMaxScale = Scale()
```

Comment:
are you sure about this?
this `val` instance is added to all elements during creation (everywhere)
if you want to change scale of prepared data you should call `setMultiplier` on this instance

Reply:
`Scale(shift)` (default 0.0)
We have to change "shift" value for sensitivity curve (100.0 if alone or with deviation slope on a graph to read percentage in scale or 0.0 if it is with other curves)